### PR TITLE
Add A and O with diaereses to CP1252 button font

### PR DESCRIPTION
**Pull request by zenseii**
Needed for Swedish.
image
image

### CODE REVIEW

**Review by ihhub — APPROVED** [review]

File: src/fheroes2/gui/ui_font.cpp [hunk 1]
```diff
@@ -5581,9 +5581,18 @@ namespace
         // Change spacing relative to other characters.
         released[192 - 32].setPosition( buttonFontOffset.x - 1, -3 );
         released[193 - 32].setPosition( buttonFontOffset.x - 1, -3 );
+        released[196 - 32].setPosition( buttonFontOffset.x - 2, -2 );
```
Comment (Districh-ru) [inline src/fheroes2/gui/ui_font.cpp:5584]: The original "A" letter has '-1' horizontal offset, while here "Ä" has '-2'.
https://github.com/ihhub/fheroes2/blob/3a0474c6f70401855e5069c6ff2287545973aba1/src/fheroes2/gui/ui_font.cpp#L4301

If '-2' better fits the 'A' letter then we should change offsets for all "A" letters.

Also for CP1250 there is no horizontal offset for all "A" letters so later we should put there '-1' or '-2'.

**Review by Districh-ru — CHANGES_REQUESTED** [review]
Hi, @zenseii. I left one question. Could you take a look when you have time.

**Review by Districh-ru — APPROVED** [review]
👍 
OK, @zenseii, if it will be needed we will update "A" offsets for other code pages.

**Comment by ihhub** [comment]
@zenseii , thank you so much for this change!

**Pull request merged**